cnivpp/localdb.go

```diff
@@ -51,8 +52,9 @@ const debugVppDb = false
 // This structure is a union of all the VPP data (for all types of
 // interfaces) that need to be preserved for later use.
 type VppSavedData struct {
-	SwIfIndex     uint32 `json:"swIfIndex"`     // Software Index, used to access the created interface, needed to delete interface.
-	MemifSocketId uint32 `json:"memifSocketId"` // Memif SocketId, used to access the created memif Socket File, used for debug only.
+	l2SwIfIndex        interface_types.InterfaceIndex         `json:"swIfIndex"`
```

Comment:
do we need both `l2SwIfIndex` and `interfaceSwIfIndex` ? if they are the same thing?

Reply:
done